app/components/courses/summary_card_component.rb

```diff
@@ -24,37 +23,23 @@ def title
       end
     end
 
-    def location_key
-      t(".location_key.#{course.funding}", count: @available_placements_count)
-    end
-
     def location_value
       return t('.location_value.not_listed') if @available_placements_count.zero?
 
-      if search_by_location?
-        t(
-          '.location_value.distance',
-          school_term:,
-          distance: content_tag(:span, pluralize(course.minimum_distance_to_search_location.ceil, 'mile'), class: 'govuk-!-font-weight-bold'),
-          location: content_tag(:span, sanitize(@location), class: 'govuk-!-font-weight-bold')
-        ).html_safe
-      else
-        t(
-          '.location_value.potential_schools',
-          count: @available_placements_count,
-          school_term:
-        )
-      end
-    end
-
-    def location_hint
       return unless search_by_location?
 
       t(
-        '.location_value.distance_hint_html',
+        '.location_value.distance',
         school_term:,
-        count: @available_placements_count
-      )
+        distance: content_tag(:span, pluralize(course.minimum_distance_to_search_location.ceil, 'mile'), class: 'govuk-!-font-weight-bold'),
+        location: content_tag(:span, sanitize(@location), class: 'govuk-!-font-weight-bold')
+      ).html_safe
+    end
+
+    def location_hint
+      return if search_by_location?
+
+      t('.location_value.placement_hint_html', school_term:)
```

Comment:
Need to dive deep more, but after this changes is @available_placements_count used anywhere? Does it contain the same meaning or needs deletion?